Whiskey/Functions/Uninstall-WhiskeyNuGetPackage.ps1

```diff
@@ -0,0 +1,45 @@
+function Uninstall-WhiskeyNuGetPackage
+{
+    <#
+    .SYNOPSIS
+    Removes a NuGet package.
+
+    .DESCRIPTION
+    The `Uninstall-WhiskeyNuGetPackage` removes a NuGet package from the `packages` directory in your build root. Pass the name of the package to the `Name` parameter. By default, all versions of a package are removed. If you want to uninstall a specific version, pass that version to the `Version` parameter. Wildcards supported. Pass the path to your build root (the directory where your packages directory is located) to the `BuildRoot` parameter.
+
+    .EXAMPLE

```

Comment:
Add `.EXAMPLE` before the first and second examples. Add a description after each example that explains what the example is demonstrating, e.g. "This example demonstrates...".